lib/helper/difference.helper.js

```diff
@@ -0,0 +1,6 @@
+function difference(arr1, arr2) {
```

Comment:
Ofwel best verwijzen naar docs:

```suggestion
// See https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_difference
function difference(arr1, arr2) {
```

Reply:
zie vorige